lib/actions-oclif/device/index.ts

```diff
@@ -30,6 +30,15 @@ interface ExtendedDevice extends DeviceWithDeviceType {
 	device_type?: string;
 	commit?: string;
 	last_seen?: string;
+	memory_usage_mb?: number;
```

Comment:
`mb` → `mib`
I gather it is [mib](https://en.wikipedia.org/wiki/Mebibyte) rather than [mb](https://en.wikipedia.org/wiki/Megabyte), right? That's what I heard in the release party live stream. :-) It's very good to have a unit of measurement as part of the field name, 👍   but having an incorrect unit is less helpful.

Reply:
These vars have been created just so we can rename the headers in the table. Unfortunately we are stuck with caps, so it will be displayed as MIB, which people may not recognise as a unit.